pkgs/applications/networking/mailreaders/thunderbird/default.nix

```diff
@@ -7,7 +7,7 @@
 , autoconf213, which, m4
 , writeScript, xidel, common-updater-scripts, coreutils, gnused, gnugrep, curl
 , enableGTK3 ? false, gtk3, wrapGAppsHook
-, debugBuild ? false
+, debugBuild ? false, config
```

Comment:
Force-push is usually okay (and we'll need to squash it in the end anyway). Thanks!